ui/components/multichain/account-details/account-details.js

```diff
@@ -91,47 +92,49 @@ export const AccountDetails = ({ address }) => {
           >
             {attemptingExport ? t('showPrivateKey') : avatar}
           </ModalHeader>
-          {attemptingExport ? (
-            <>
-              <Box
-                display={Display.Flex}
-                alignItems={AlignItems.center}
-                flexDirection={FlexDirection.Column}
-              >
-                {avatar}
-                <Text
-                  marginTop={2}
-                  marginBottom={2}
-                  variant={TextVariant.bodyLgMedium}
-                  style={{ wordBreak: 'break-word' }}
+          <ModalBody>
+            {attemptingExport ? (
+              <>
+                <Box
+                  display={Display.Flex}
+                  alignItems={AlignItems.center}
+                  flexDirection={FlexDirection.Column}
                 >
-                  {name}
-                </Text>
-                <AddressCopyButton address={address} shorten />
-              </Box>
-              {privateKey ? (
-                <AccountDetailsKey
-                  accountName={name}
-                  onClose={onClose}
-                  privateKey={privateKey}
-                />
-              ) : (
-                <AccountDetailsAuthenticate
-                  address={address}
-                  onCancel={onClose}
-                  setPrivateKey={setPrivateKey}
-                  setShowHoldToReveal={setShowHoldToReveal}
-                />
-              )}
-            </>
-          ) : (
-            <AccountDetailsDisplay
-              accounts={accounts}
-              accountName={name}
-              address={address}
-              onExportClick={() => setAttemptingExport(true)}
-            />
-          )}
+                  {avatar}
+                  <Text
+                    marginTop={2}
+                    marginBottom={2}
+                    variant={TextVariant.bodyLgMedium}
+                    style={{ wordBreak: 'break-word' }}
+                  >
+                    {name}
+                  </Text>
+                  <AddressCopyButton address={address} shorten />
+                </Box>
+                {privateKey ? (
+                  <AccountDetailsKey
+                    accountName={name}
+                    onClose={onClose}
+                    privateKey={privateKey}
+                  />
+                ) : (
+                  <AccountDetailsAuthenticate
+                    address={address}
+                    onCancel={onClose}
+                    setPrivateKey={setPrivateKey}
+                    setShowHoldToReveal={setShowHoldToReveal}
+                  />
+                )}
+              </>
+            ) : (
+              <AccountDetailsDisplay
+                accounts={accounts}
+                accountName={name}
+                address={address}
+                onExportClick={() => setAttemptingExport(true)}
+              />
+            )}
+          </ModalBody>
```

Comment:
Wraps modal content in `ModalBody` which handles the padding of the modal content. component library docs have been updated https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-modal--docs

<img width="1511" alt="Screenshot 2024-02-01 at 9 06 50 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/1e308ade-d277-4f48-a00f-5b05a2ede1e5">